addon/helpers/choice.js

```diff
@@ -9,6 +9,11 @@ export default MicroState.extend({
       return Type.create(values, options);
     },
 
+    set(choice, values, options) {
```

Comment:
If they wanted to yes, or a JSON structure that the `set` transition could use. I don't see it as a vey common use-case for `choice`